v-next/hardhat/src/internal/builtin-plugins/network-manager/edr/utils/convert-to-edr.ts

```diff
@@ -102,49 +152,6 @@ export function hardhatHardforkToEdrSpecId(hardfork: string): string {
   }
 }
 
-export function edrSpecIdToHardhatHardfork(specId: string): HardforkName {
```

Comment:
We don’t seem to use this anywhere. Maybe it was left behind during a refactor that removed the calls?

Reply:
Seems safe to remove